src/azure-cli/azure/cli/command_modules/cdn/_help.py

```diff
@@ -102,6 +102,12 @@
     text: >
         az cdn endpoint create -g group -n endpoint --profile-name profile
         --origin www.example.com 88 4444
+  - name: Create an endpoint with a custom domain origin with private link enabled.
+    text: >
+        az cdn endpoint create -g group -n endpoint --profile-name profile
+        --origin www.example.com 80 443
+        /subscriptions/subid/resourcegroups/rg1/providers/Microsoft.Network/privateLinkServices/pls1
```

Comment:
One more question, can we have extra three arguments instead of combining all these into origin?

Reply:
An upcoming feature for CDN adds support for multiple origins, so the private link details must be specified for particular origins. I agree that this isn't an ideal solution, but is was the only way i could figure without breaking changes. If this is a hard blocker, we can just remove the ability to enable private link on origins via this command, and require users to set private link details via `az cdn origin update` and the soon-to-be-added `az cdn origin create`.